### DISCUSSION TITLE
Pylance

### DISCUSSION THREAD

**Discussion by mauricekraus** [askfiy/nvim#27] [Q&A] [answered]
Hey @askfiy , 

i saw that you tried to implement pylance, why did you remove it? It would be awesome to use the much more powerful pylance instead of pyright. I tried to search online, but i did not find anybody using it in neovim, which prob means it doesn't work.
Thanks anyway, awesome project!

**Comment by askfiy** [comment, marked as answer]
pylance It can be used in nvim. But it needs to modify some code of pylance. I think this is inconvenient for most people, so this project still uses pyright by default. But I will still use pylance privately. If you are willing to study it, you can try to modify the code of pylance. Although it is encrypted, the way is very simple.

**Reply by mauricekraus** [reply]
Thank you.

**Reply by Dave-Elec** [reply]
I wanted to try it out. Do you have any directions on how to make it work?

**Reply by dpetka2001** [reply]
Hi @askfiy I have managed to install pylance privately but am wondering about code actions that it offers in Neovim. For me it only offers `move symbol to` code actions. No code actions for auto import like in vscode. Do you also have the same problem? Or is it something I missed in the configuration of pylance on my end?

**Reply by askfiy** [reply]
> Hi @askfiy I have managed to install pylance privately but am wondering about code actions that it offers in Neovim. For me it only offers `move symbol to` code actions. No code actions for auto import like in vscode. Do you also have the same problem? Or is it something I missed in the configuration of pylance on my end?

I rarely use vscode, but I often use auto import for quick fixes.
